ui/packages/catalog-ui-search/src/main/webapp/component/sort-item/sort-item.collection.view.js

```diff
@@ -19,19 +19,29 @@ define([
     'jquery',
     'js/CustomElements',
     './sort-item.view',
+    './sort-item-collection.hbs',
     'js/store'
-], function (Marionette, _, $, CustomElements, queryItemView, store) {
+], function (Marionette, _, $, CustomElements, queryItemView, template, store) {
 
-    return Marionette.CollectionView.extend({
+    return Marionette.CompositeView.extend({
```

Comment:
Why is this a composite view?

Reply:
I wanted to keep the `add` button with the collection, instead of having to declare it twice. I think I can do the same thing with something else though, since `CompositeView` is deprecated in later versions.